### PR TITLE
refactor(finance): rename account to accountNumber

### PR DESCRIPTION
**Pull request by Shinigami92**
references https://github.com/faker-js/faker/issues/1346#issuecomment-1407724091

### CODE REVIEW

File: src/modules/finance/index.ts [hunk 1]
```diff
@@ -59,21 +60,45 @@ export class FinanceModule {
...
+  /**
+   * Generates a random account number.
+   *
+   * @param length The length of the account number. Defaults to `8`.
+   *
+   * @example
+   * faker.finance.accountNumber() // 92842238
+   * faker.finance.accountNumber(5) // 32564
+   *
+   * @since 2.0.1
```
Comment (Shinigami92) [inline src/modules/finance/index.ts]:
```suggestion
   * @since 8.0.0
```

File: src/modules/finance/index.ts [hunk 2]
```diff
@@ -59,21 +60,45 @@ export class FinanceModule {
...
    * @param options An options object. Defaults to `{}`.
    * @param options.length The length of the account number. Defaults to `8`.
    *
    * @example
-   * faker.finance.account() // 92842238
-   * faker.finance.account({ length: 5 }) // 32564
+   * faker.finance.accountNumber() // 92842238
+   * faker.finance.accountNumber({ length: 5 }) // 32564
    *
    * @since 2.0.1
```
Comment (xDivisionByZerox) [inline src/modules/finance/index.ts]:
```suggestion
   * @since 8.0.0
```

File: src/modules/finance/index.ts [hunk 3]
```diff
@@ -88,13 +113,13 @@ export class FinanceModule {
...
    *
    * @example
-   * faker.finance.account() // 92842238
-   * faker.finance.account(5) // 28736
-   * faker.finance.account({ length: 5 }) // 32564
+   * faker.finance.accountNumber() // 92842238
+   * faker.finance.accountNumber(5) // 28736
+   * faker.finance.accountNumber({ length: 5 }) // 32564
    *
    * @since 2.0.1
```
Comment (xDivisionByZerox) [inline src/modules/finance/index.ts]:
```suggestion
   * @since 8.0.0
```

**Review by xDivisionByZerox — CHANGES_REQUESTED** [review]
Please include this in the upgrading guide.

File: src/modules/finance/index.ts [hunk 4]
```diff
@@ -59,21 +60,45 @@ export class FinanceModule {
    * faker.finance.account(5) // 32564
    *
    * @since 2.0.1
+   *
+   * @deprecated Use `faker.finance.accountNumber` instead.
```
Comment (ST-DDT) [inline src/modules/finance/index.ts:66]: This should also have a `@see` tag.

**Review by ST-DDT — DISMISSED** [review]

**Review by xDivisionByZerox — DISMISSED** [review]

File: test/finance.spec.ts [hunk 5]
```diff
@@ -114,26 +118,58 @@ describe('finance', () => {
...
           expect(
-            accountNum,
+            accountNumber,
             'The length of the account number should be 8 characters long'
           ).toHaveLength(8);
         });
 
-        it('should be the the length fo given number', () => {
-          const accountNum = faker.finance.account(16);
+        it('should be the the length to given number', () => {
```
Comment (ejcheng) [inline test/finance.spec.ts]:
```suggestion
        it('should have same length as given length number', () => {
```

File: test/finance.spec.ts [hunk 6]
```diff
@@ -114,26 +118,58 @@ describe('finance', () => {
...
+          const accountNumber = faker.finance.accountNumber();
+
+          expect(accountNumber).toBeTruthy();
+          expect(
+            accountNumber,
+            'The length of the account number should be 8 characters long'
+          ).toHaveLength(8);
+        });
+
+        it('should be the the length to given number', () => {
```
Comment (ejcheng) [inline test/finance.spec.ts]:
```suggestion
        it('should have same length as given length number', () => {
```

File: test/finance.spec.ts [hunk 7]
```diff
@@ -114,26 +118,58 @@ describe('finance', () => {
...
+          const accountNumber = faker.finance.accountNumber(16);
+
+          expect(accountNumber).toBeTruthy();
+          expect(
+            accountNumber,
+            'The length of the account number should match the given number'
+          ).toHaveLength(16);
+        });
+
+        it('should be the the length option to given number', () => {
```
Comment (ejcheng) [inline test/finance.spec.ts]:
```suggestion
        it('should have same length as given length object', () => {
```

**Review by ejcheng — CHANGES_REQUESTED** [review]

**Review by ST-DDT — APPROVED** [review]

**Review by ejcheng — APPROVED** [review]

**Review by matthewmayer — APPROVED** [review]

**Pull request merged**